api/types/src/headers.rs

```diff
@@ -17,3 +17,5 @@ pub const X_APTOS_OLDEST_BLOCK_HEIGHT: &str = "X-Aptos-Oldest-Block-Height";
 pub const X_APTOS_LEDGER_TIMESTAMP: &str = "X-Aptos-Ledger-TimestampUsec";
 /// Cursor used for pagination.
 pub const X_APTOS_CURSOR: &str = "X-Aptos-Cursor";
+/// Included in requests from clients we provide to identify the request source.
+pub const X_APTOS_SOURCE_CLIENT: &str = "X-Aptos-Source-Client";
```

Comment:
How is this different than the User-Agent header?  Should get that for free automatically from SDKs and others, and is more common across sources.

Reply:
We spoke offline, we'll use User-Agent. See https://github.com/aptos-labs/aptos-core/pull/7061 for more discussion on this topic.